data/resources/style.css

```diff
@@ -150,9 +170,13 @@ messagebubble:not(.outgoing):dir(rtl) {
   margin-left: 38px;
 }
 
+messagebubble.outgoing messagereply {
+  color: @accent_color;
+}
```

Comment:
This was not done on purpose because it's difficult to achieve a good contrast with the colored background of the outgoing bubbles. It's done in the same way in other clients.